platform/kobo/disable-wifi.sh

```diff
@@ -40,11 +40,11 @@ ifconfig "${INTERFACE}" down
 
 # Some sleep in between may avoid system getting hung
 # (we test if a module is actually loaded to avoid unneeded sleeps)
-if grep -q "${WIFI_MODULE}" "/proc/modules"; then
+if grep -q "^${WIFI_MODULE}" "/proc/modules"; then
     usleep 250000
     rmmod "${WIFI_MODULE}"
 fi
-if grep -q "sdio_wifi_pwr" "/proc/modules"; then
+if grep -q "^sdio_wifi_pwr" "/proc/modules"; then
```

Comment:
What does start of line have to do with whether it's loaded?

Reply:
Poor choice of words ;).

Module dependencies are listed after the first entry, we don't want to match on those (although it's technically not an issue here).